docs/source/entities-advanced.mdx

```diff
@@ -1025,3 +1031,54 @@ type C @key(fields: "id") {
 ```
 
 In a more complex graph, a field could be reachable via multiple paths, and there could be a different field used to resolve the `prop` depending on which path was used.
+
+### Referencing fields across subgraphs
+
+The definition of context scopes can only exist in one subgraph schema. You can't define `@context` in one subgraph and use `@fromContext` in another. However, you can use contexts to share data across subgraphs using the `@external` reference.
```

Comment:
Rather than "You can't define `@context` in one subgraph and use `@fromContext` in another."

Maybe say:
"`@fromContext` cannot reference a `@context` defined in another subgraph"